app/views/monthlyContributions.scala.html

```diff
@@ -33,9 +34,13 @@
         };
         window.guardian.payPalType = @payPalButton? 'ExpressCheckout' : 'NotSet';
         window.guardian.directDebitEnable = @directDebitButtonEnable;
-        window.guardian.stripeKey = {
-          default: "@defaultStripeConfig.publicKey",
-          uat: "@uatStripeConfig.publicKey"
+        window.guardian.stripeKeyRestOfTheWorld = {
+          default: "@defaultStripeConfig.forCountry(None).publicKey",
+          uat: "@uatStripeConfig.forCountry(None).publicKey"
+        };
+        window.guardian.stripeKeyAustralia = {
+          default: "@defaultStripeConfig.forCountry(Some(Country.Australia)).publicKey",
```

Comment:
You could `import Country.Australia` then you can just put `Some(Australia)` here